regtests/client/python/cli/command/__init__.py

```diff
@@ -86,7 +94,9 @@ def options_get(key, f=lambda x: x):
                 catalog_name=options_get(Arguments.CATALOG),
                 catalog_role_name=options_get(Arguments.CATALOG_ROLE),
                 principal_role_name=options_get(Arguments.PRINCIPAL_ROLE),
-                properties=properties
+                properties={} if properties is None else properties,
+                set_properties={} if set_properties is None else set_properties,
+                remove_properties=[] if remove_properties is None else remove_properties
             )
         elif options.command == Commands.PRIVILEGES:
             from cli.command.privileges import PrivilegesCommand
```

Comment:
the namespaces command seems to be missing the `set_properties` + `remove_properties `changes

Reply:
So when I went in to make all the commands consistent, I discovered that `namespaces.py` actually doesn't implement `update` at all yet. And since the protocol is actually a bit different, I think we could have a separate PR to better address "missing" features of the Catalogs API (or make a decision to actually pull in something from pyiceberg or similar).

So the fixes for `namespaces.py` right now are just to fix the behavior of existing functionality.

Reply:
That makes sense, let's just leave it out for now then